index.html

```diff
@@ -87,6 +87,14 @@
       <div class="medal">
         <a href='https://operationcode.org'><img src="images/medal_navy.png" alt="opcodelogo"></a>
       </div>
+      <ul class="supporters">
```

Comment:
You asked me about an image size. The OC image? It is a little blurry. You can resize it, or make it an SVG.